index.php

```diff
@@ -1,45 +1,55 @@
 <?php
 // показывать или нет выполненные задачи
-$show_complete_tasks = rand(0, 1);
-$projects_categ_arr = ['Входящие', 'Учеба', 'Работа', 'Домашние дела', 'Авто'];
-$tasks_list_arr = [
+$showCompleteTasks = rand(0, 1);
+$projectsCategArr = ['Входящие', 'Учеба', 'Работа', 'Домашние дела', 'Авто'];
+$tasksListArr = [
     [
         'task' => 'Собеседование в IT компании',
-        'finish_date' => '01.12.2019',
+        'finishDate' => '01.12.2019',
         'category' => 'Работа',
-        'finish_flag' => false,
+        'finishFlag' => false,
     ],
     [
         'task' => 'Выполнить тестовое задание',
-        'finish_date' => '25.12.2019',
+        'finishDate' => '25.12.2019',
         'category' => 'Работа',
-        'finish_flag' => false,
+        'finishFlag' => false,
     ],
     [
         'task' => 'Сделать задание первого раздела',
-        'finish_date' => '21.12.2019',
+        'finishDate' => '21.12.2019',
         'category' => 'Учеба',
-        'finish_flag' => true,
+        'finishFlag' => true,
     ],
     [
         'task' => 'Встреча с другом',
-        'finish_date' => '22.12.2019',
+        'finishDate' => '22.12.2019',
         'category' => 'Входящие',
-        'finish_flag' => false,
+        'finishFlag' => false,
     ],
     [
         'task' => 'Купить корм для кота',
-        'finish_date' => null,
+        'finishDate' => null,
         'category' => 'Домашние дела',
-        'finish_flag' => false,
+        'finishFlag' => false,
     ],
     [
         'task' => 'Заказать пиццу',
-        'finish_date' => null,
+        'finishDate' => null,
         'category' => 'Домашние дела',
-        'finish_flag' => false,
+        'finishFlag' => false,
     ],
 ];
+
+function countTasks($tasksLists, $projectName) {
```

Comment:
Под типизацией имею в виду казать типы аргументов и тип отдаваемого результата. В твоём случае 
```suggestion
function countTasks(array $tasksList, string $projectName): int
{
```

Reply:
сделал